app/dts/bindings/behaviors/zmk,behavior-macro-press.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: Macro Press Mode Behavior
+
+compatible: "zmk,behavior-macro-press"
```

Comment:
This won't do anything if you try to put it directly in a keymap's bindings, so I wonder if calling it a "behavior" might be confusing. Should we come up with a new name for these? Maybe something like "behavior modifier" or "behavior config"?

Reply:
How about `zmk,macro-control-press` ?

Reply:
That seems pretty good to me.

Reply:
I went with `zmk,macro-control-mode-press`.